web/utils.py

```
@@ -263,7 +263,9 @@ def get_leaderboard(current_user=None, period=None, limit=10):
     if period == "weekly":
         # Get weekly leaderboard
         leaderboard_entries = (
-            Points.objects.filter(awarded_at__gte=one_week_ago, user__profile__is_teacher=False)
+            Points.objects.filter(
+                awarded_at__gte=one_week_ago, user__profile__is_teacher=False, user__profile__is_profile_public=True
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)